ui/src/main/resources/EntraID/Code/EntraOIDCClientConfigurationSheet.xml

```diff
@@ -0,0 +1,657 @@
+        <largeStorage>0</largeStorage>
+        <multiSelect>0</multiSelect>
+        <name>cache</name>
+        <number>5</number>
+        <prettyName>Caching policy</prettyName>
+        <relationalStorage>0</relationalStorage>
+        <separator> </separator>
+        <separators>|, </separators>
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>long|short|default|forbid</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </cache>
+      <code>
+        <contenttype>PureText</contenttype>
+        <disabled>0</disabled>
+        <editor>PureText</editor>
+        <name>code</name>
+        <number>2</number>
+        <prettyName>Code</prettyName>
+        <restricted>0</restricted>
+        <rows>20</rows>
+        <size>50</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.TextAreaClass</classType>
+      </code>
+      <name>
+        <disabled>0</disabled>
+        <name>name</name>
+        <number>1</number>
+        <prettyName>Name</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </name>
+      <parse>
+        <disabled>0</disabled>
+        <displayFormType>select</displayFormType>
+        <displayType>yesno</displayType>
+        <name>parse</name>
+        <number>4</number>
+        <prettyName>Parse content</prettyName>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      </parse>
+      <use>
+        <cache>0</cache>
+        <disabled>0</disabled>
+        <displayType>select</displayType>
+        <freeText>forbidden</freeText>
+        <largeStorage>0</largeStorage>
+        <multiSelect>0</multiSelect>
+        <name>use</name>
+        <number>3</number>
+        <prettyName>Use this extension</prettyName>
+        <relationalStorage>0</relationalStorage>
+        <separator> </separator>
+        <separators>|, </separators>
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>currentPage|onDemand|always</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </use>
+    </class>
+    <property>
+      <cache>long</cache>
+    </property>
+    <property>
+      <code>define('entra-properties', {
+  prefix: 'entra.properties.data.',
+  keys: [
+    'newEntry',
+    'tip.key.groupMap',
+    'tip.value.groupMap',
+    'tip.key.userMap',
+    'tip.value.userMap',
+    'tip.value.forbiddenGroup',
+    'tip.value.allowedGroup',
+  ]
+});
+require.config({
+  paths: {
+    'xwiki-suggestGroups': "$xwiki.getSkinFile('uicomponents/suggest/suggestUsersAndGroups.js', true)" +
+      "?v=$escapetool.url($xwiki.version)"
+  }
+});
+
+
+require(['jquery',  'xwiki-l10n!entra-properties', 'xwiki-suggestGroups'],function($, l10n) {
+  $(document).ready(function() {
```

Comment:
Also, I think it would be even nicer to have methods declarations outside the `$(function() {})`, so to have better visibility on what is exactly executed when document is ready. But it might be a personal choice, it's not affecting the functionality